minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -62,16 +63,19 @@
 
   protected String mWorkDirectory;
   protected String mHostname;
+  protected JournalType mJournalType;
```

Comment:
I don't think we need this, we already have `ServerConfiguration` giving us the journal type

Reply:
This is because some tests do not support embedded journal. For those test, we allow them to set journal type. 
but the `journal type in ServerConfiguration` may also not be correct, it could be set by previous test and we do not reset!

I may think about adding a `default test journal type` property or make sure `journal type in ServerConfiguration` is reset after every test.

Reply:
if a test doesn't support embedded journal, should it explicitly configure itself to use ufs journal?

I think we already have this line to reset configuration after each test: https://github.com/Alluxio/alluxio/blob/master/minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java#L199

Reply:
Yeah! In both situations, the configuration will be reset or reload the system property. The master journal type will be consistent through tests. Thanks!